contrib/cirrus/lib.sh

```diff
@@ -349,6 +349,20 @@ remove_packaged_podman_files(){
     done
 }
 
+systemd_banish(){
+    echo "Disabling periodic services that could destabalize testing:"
+    set +e  # Not all of these exist on every platform
+    for unit in cron atd apt-daily-upgrade apt-daily fstrim motd-news systemd-tmpfiles-clean
```

Comment:
It's called `crond` on Fedora. Might as well disable it there too.

Reply:
Ohh good catch, ya, I'll do that.